Configs/Defaults.plist

```diff
@@ -13,7 +13,7 @@
 	<key>CFBundlePackageType</key>
 	<string>FMWK</string>
 	<key>CFBundleShortVersionString</key>
-	<string>2.0.2</string>
+	<string>$(MARKETING_VERSION)</string>
```

Comment:
Can you remove all the other unrelated changes like this one?

Reply:
```suggestion
	<string>2.0.2</string>
```